en/docs/learn/tibco-migration-assistant.md

```diff
@@ -0,0 +1,192 @@
+# TIBCO BusinessWorks Migration
+This guide explains how to use the [migrate-tibco](https://central.ballerina.io/wso2/tool_migrate_tibco/latest) tool to convert
+[TIBCO BusinessWorks](https://docs.tibco.com/products/tibco-activematrix-businessworks) integrations into Ballerina packages compatible with the [WSO2 Ballerina Integrator](https://wso2.com/integrator/ballerina-integrator).
+## Tool Overview
+
+he tool accepts either a BusinessWorks project directory or a standalone `bwp` file as input and generates an equivalent Ballerina package that can be opened in WSO2 Ballerina Integrator.
+
+## Installation
+
+Execute the command below to pull the `migrate-tibco` tool from Ballerina Central
+```bash
+$ bal tool pull migrate-tibco
+```
+
+## Usage
+
+### Command Syntax
+
+```bash
+$ bal migrate-tibco <source-project-directory-or-file> [-o|--out <output-directory>]
+```
+
+### Parameters
+
+- **source-project-directory-or-file** - Required. The TIBCO BusinessWorks project directory or `bwp` file to migrate.
+- **-o or --out** - *Optional*. The directory where the new Ballerina package will be created. If the directory does not exist tool will create it for you. If not provided,
+  - If source-project-directory-or-file is a directory it will create new directory named ${source-project-directory-or-file}_converted in the root of source-project-directory-or-file
+  - if source-project-directory-or-file is a file it will create a new directory named ${root}_converted in the parent of the root directory where root is the directory containing the given file.
```

Comment:
```suggestion
  - if source-project-directory-or-file is a file, it will create a new directory named ${root}_converted in the parent of the root directory where root is the directory containing the given file.
```